projects/media-viewer/src/lib/viewers/convertible-content-viewer/document-conversion-api.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable, of } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+
+@Injectable()
+export class DocumentConversionApiService {
+
+  public documentConversionUrl = '/doc-assembly/convert';
```

Comment:
why is this public? it should be a constant as well

Reply:
It says class member cannot be const, so just changed to private

Reply:
maybe set it as readonly as well